code/game/objects/structures/trash_pile.dm

```diff
@@ -0,0 +1,102 @@
+/obj/structure/trash_pile
+	name = "trash pile"
+	desc = "A heap of garbage, but maybe there's something interesting inside?"
+	icon = 'icons/obj/trash_piles.dmi'
+	icon_state = "randompile"
+	density = TRUE
+	anchored = TRUE
+	layer = TABLE_LAYER
+	obj_flags = CAN_BE_HIT
+	pass_flags = LETPASSTHROW
+
+	max_integrity = 50
+
+	var/hide_item_time = 15
```

Comment:
this doesn't really seem important for admins to be able to edit, i think it'd be better off as a define

Reply:
also, document it when you do 🤓